### DISCUSSION TITLE
Solar export setting

### DISCUSSION THREAD

**Discussion by wevcore** [kellerza/sunsynk#126] [General]
Hello, 

Please advise if there is a sensor available (or if not, the modbus register for creating a custom sensor) for switching the 'solar export' setting on/off.

Thanks, 
Kevin
![Solar export](https://user-images.githubusercontent.com/46636206/232538780-69b43f0d-1df0-433c-a49a-87ba6625cc34.PNG)

**Comment by kellerza** [comment]
load_limit might be a combination of export+work mode

Try and give some feedback

**Comment by iBeech** [comment]
I have put a pull request in which adds this sensor: https://github.com/kellerza/sunsynk/pull/196/
